scripts/tier_analyzer.py

```diff
@@ -0,0 +1,100 @@
+#!/usr/bin/env python
+
+import argparse
+import re
+import subprocess
+import sys
+
+
+def check_virtualenv():
+    """Check if we are in virtualenv and if not, exit."""
+    if not hasattr(sys, 'real_prefix'):
+        print('You must activate CFME virtualenv in oder to run this script.')
+        sys.exit(1)
+
+
+def get_pytest_output(args, use_tier_marker=False):
```

Comment:
**REQUIRED:** This function is not as generic as the name implies.   Change the name to something more specific such as get_pytest_collect_only_output().